intuita-webview/src/codemodList/TreeView/TreeItem.tsx

```diff
@@ -19,23 +24,90 @@ type Props = {
 	onClick(): void;
 	depth: number;
 	disabled: boolean;
+	rootPath: string;
+	executionPath?: T.These<SyntheticError, string>;
 };
 
 const TreeItem = ({
 	id,
 	label,
 	progressBar,
 	description,
-	hoverDescription,
 	kind,
 	icon,
 	open,
 	focused,
+	rootPath,
 	actionButtons,
 	hasChildren,
 	onClick,
 	depth,
+	executionPath,
 }: Props) => {
+	const [targetPath, setTargetPath] = useState('');
```

Comment:
does this come from (or should come from) the workspace state?
Edit: I see this is a volatile path (you could even call it volatile if you want)
Edit2: but where is it set?